src/main/java/org/tikv/common/ReadOnlyPDClient.java

```diff
@@ -72,4 +72,6 @@ List<Pdpb.Region> scanRegions(
   Long getClusterId();
 
   RequestKeyCodec getCodec();
+
+  Long UpdateServiceGCSafePoint(String serviceId, long ttl, long safePoint, BackOffer backOffer);
```

Comment:
Suggest to comment about the usage of this method. Especially for the unit of `ttl`, and how to get the `safePoint`.

```suggestion
  Long updateServiceGCSafePoint(String serviceId, long ttl, long safePoint, BackOffer backOffer);
```